workflow/workflow.go

```diff
@@ -33,6 +34,11 @@ func (w *Workflow) Start() error {
 	if w.eventStream != nil {
 		return fmt.Errorf("workflow engine already running")
 	}
+	go func() {
+		for err := range w.ErrC {
+			logrus.Error(err)
```

Comment:
Do not log errors in package, `ErrC` is exposed and errors should be logged in the main.go or package for collecting and logging all errors.

Reply:
done